tests/ui/resolve/resolve-self-in-impl.stderr

```diff
@@ -1,40 +1,40 @@
 error: `Self` is not valid in the self type of an impl block
```

Comment:
For some reason my change leads to a reordering of diagnostics. Shouldn't matter too much.